src/payment/strategies/klarna/klarna-payment-strategy.spec.ts

```diff
@@ -117,13 +123,52 @@ describe('KlarnaPaymentStrategy', () => {
         });
 
         it('loads widget', () => {
+            const billingAddress = {
+                street_address: '12345 Testing Way',
+                city: 'Some City',
+                country: 'US',
+                given_name: 'Test',
+                family_name: 'Tester',
+                postal_code: '95555',
+                region: 'California',
+                email: 'test@bigcommerce.com',
+            };
+            const shippingAddress = {
+                street_address: '12345 Testing Way',
+                city: 'Some City',
+                country: 'US',
+                given_name: 'Test',
+                family_name: 'Tester',
+                postal_code: '95555',
+                region: 'California',
+                email: 'test@bigcommerce.com',
+            };
+
             expect(klarnaCredit.init).toHaveBeenCalledWith({ client_token: 'foo' });
+            expect(klarnaCredit.load).toHaveBeenCalledWith({ container: '#container' }, { billing_address: billingAddress, shipping_address: shippingAddress }, expect.any(Function) );
             expect(klarnaCredit.load).toHaveBeenCalledTimes(1);
         });
 
         it('triggers callback with response', () => {
             expect(onLoad).toHaveBeenCalledWith({ show_form: true });
         });
+
+        it('throws error if required data is not loaded', async () => {
```

Comment:
Not sure if this is the best way to verify error throws. Have you tried with jest's `toThrowError`?

Reply:
I guess in case of this test, is good to go with Sonora's approach since `strategy.initialize` is a promise and needs to be resolved (Maybe we can check if toThrowError supports promise response